environment.yml

```diff
@@ -0,0 +1,18 @@
+name: UPenn_Holdings_Project
+dependencies:
+- anaconda::beautifulsoup4=4.6.0
+- anaconda::ipython=5.3.0
+- anaconda::pandas=0.20.1
+- anaconda::pep8=1.7.0
+- anaconda::pip=9.0.1
+- anaconda::python=3.6.1
+- anaconda::r-base=3.4.1
+- anaconda::requests=2.14.2
+- anaconda::spyder=3.1.4
+- anaconda::sqlalchemy=1.1.9
+- anaconda::sqlite=3.13.0
+- anaconda::yaml=0.1.6
+- pip:
+  - backoff==1.4.3
+  - ipython-genutils==0.2.0
```

Comment:
What's this for? [PyPI package](https://pypi.python.org/pypi/ipython_genutils) says:

> Pretend this doesn’t exist. Nobody should use it.

Reply:
Removed in b3e7ebca. Conda added it when I originally generated the `environment` file, and I didn't notice it to delete it.